### DISCUSSION TITLE
issue with the button

### DISCUSSION THREAD

**Discussion by kpqq** [maximmax42/Discord-CustomRP#245] [Q&A] [answered]
the button dose not show up anymore in my discord profile
![fNPY3cWwlp](https://github.com/maximmax42/Discord-CustomRP/assets/143846691/9d09b1c3-50a2-4118-ad57-a294e16931c9)

**Comment by maximmax42** [comment, marked as answer]
This is a known issue, but it's not an issue with CustomRP, but with Discord. Buttons do actually display to others and work as usual.

**Reply by kpqq** [reply]
ok Thank you very much for your quick response... <3
